nncf/experimental/common/tensor_statistics/collectors.py

```diff
@@ -299,9 +306,9 @@ def register_inputs(self, inputs: Dict[int, List[NNCFTensor]]) -> None:
         for reducer in self._reducers:
             reducer_hash = hash(reducer)
             input_ = inputs[reducer_hash]
-            if any(tensor.is_empty() for tensor in input_):
-                continue
-            reduced_inputs[reducer_hash] = reducer(input_)
+            reduced_input = reducer(input_)
+            if reduced_input:
+                reduced_inputs[reducer_hash] = reduced_input
```

Comment:
Is it possible that an reducer returns value: bool(value) == False, but this values should be aggregated somehow?

Reply:
I would add a more transparent check to `None`, thanks.

Reply:
That would be great, but is it possible for some TensorCollector to somehow collect `None`s? I mean, could `None` be a valid output of a reducer? Previously we assumed reducer could not reduce empty tensors, so now I'm not sure that "reducers return None only in case aggregation should be skipped" assumption will last